README.md

```diff
@@ -32,3 +33,53 @@ class AppKernel
 #Usages
 
 You just have to continue to use former url generation, nothing changes concerning the implementation. The only changes is that you don't have to repeat current route parameters anymore
+
+##Example 1
+
+Current route:
+```yml
+app_products_show:
+    path: /shops/{shop}/products/{products}/
+```
+
+It's my current url is `/shops/12/products/345/`, I want to rebuild the same url, why should I repeat route parameters if it's my current route ? 
```

Comment:
My current url is `/shops/12/products/345/`. I want to build again the same url. Should I repeat route parameters if it already is my current route?